EECS3311-PD1/src/main/java/UI/Main.java

```diff
@@ -1,21 +1,47 @@
 package UI;
 
+import UI.Login.LoginView;
 import javafx.application.Application;
+import javafx.event.Event;
 import javafx.scene.Scene;
-import javafx.scene.control.Label;
-import javafx.scene.layout.StackPane;
 import javafx.stage.Stage;
 
 public class Main extends Application {
+    private Stage stage;
+
```

Comment:
Here you are creating the login view checking to see if the login was successful and then creating the stage which you will be showing the loginView; the loginView is the scene for the stage you are showing.